sms-authenticator/pom.xml

```diff
@@ -12,18 +12,25 @@
         <version>25.0.1</version>
     </parent>
 
+	<properties>
+		<maven.shade.version>3.5.1</maven.shade.version>
+	</properties>
+
     <dependencies>
 		<dependency>
 			<groupId>org.keycloak</groupId>
 			<artifactId>keycloak-quarkus-server-app</artifactId>
-			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>com.googlecode.libphonenumber</groupId>
+			<artifactId>libphonenumber</artifactId>
+			<version>8.13.40</version>
 		</dependency>
 		<!-- Necessary for proper execution of IDELauncher -->
 		<!-- Can be removed as part of the https://github.com/keycloak/keycloak/issues/22455 enhancement -->
 		<dependency>
 			<groupId>io.quarkus</groupId>
 			<artifactId>quarkus-vertx-http-dev-ui-resources</artifactId>
-			<scope>provided</scope>
```

Comment:
Please leave the scopes in sms-auth module.

In my original request I meant to remove scopes only from dependency management, which sits in the parent pom in the project root folder.

In summary:
- pom.xml -> no scopes
- sms-authenticator/pom.xml -> scopes